### DISCUSSION TITLE
Sophos UTM logs to SO syslog

### DISCUSSION THREAD

**Discussion by xpowershellx** [Security-Onion-Solutions/securityonion#3906] [Unsupported Versions]
I enabled sending syslog messages port 514 to SO and the logs are received but the "message" field is not parsed . How can I parse/process the message field? In Graylog which i curently use i successfully did it with extractors .Thanks

**Comment by weslambert** [comment]
We are moving towards using Filebeat modules to assist with logs like Sophos UTM logs and others.  In the time being, you can create your own ingest node pipeline in `/opt/so/saltstack/local/salt/elasticsearch/files/ingest` (also copying and modifying the `syslog` pipeline from the equivalent `default` directory as needed).

**Reply by corycandia** [reply]
> We are moving towards using Filebeat modules to assist with logs like Sophos UTM logs and others

@weslambert Do you guys have a road map or timeline for this?  

I know we all want you guys to do everything imaginable feature wise, have it by tomorrow, and for free (note the sarcasm and appreciation for such amazing work on SO by you guys), but I'm curious as I am also very much interested in getting some in depth ingestion of specific network device data like the UTM or other, as mentioned in [this out post](https://github.com/Security-Onion-Solutions/securityonion/discussions/5173#discussioncomment-1209383).  The amount of time I have to wait might dictate how much of my life I'm about to invest in learning how to write custom ingest node configurations.
